src/styles/card-block.css

```diff
@@ -0,0 +1,92 @@
+
+.card {
+  display: flex;
+  flex-direction: column ;
+  width: 200px;
+  padding: 16px;
+
+  background: var(--white);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 16px auto 40px;
+}
+
+.card__item-name {
+  margin-bottom: 4px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--colorDark);
+}
+
+.card__product-code {
+  margin-bottom: 16px;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--colorLight);
+}
+
+.card__price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-top: 24px;
```

Comment:
I think it's better to use always margin-bottom (or margin-top) for all elements

Reply:
check @IrynaBalandiukh comment